libs/components/forms/src/lib/modules/radio/radio.component.ts

```diff
@@ -287,41 +316,41 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
 
   public setGroupDisabledState(isDisabled: boolean) {
     this.radioGroupDisabled = isDisabled;
-    this.changeDetector.markForCheck();
+    this.#changeDetector.markForCheck();
   }
 
   public registerOnChange(fn: any): void {
-    this.onChangeCallback = fn;
+    this.#onChangeCallback = fn;
   }
 
   public registerOnTouched(fn: any): void {
-    this.onTouchedCallback = fn;
+    this.#onTouchedCallback = fn;
   }
 
   public onInputChange(event: Event): void {
     event.stopPropagation();
 
     if (!this.disabled) {
       this.checked = true;
-      this._change.emit({
+      this.#_change.emit({
```

Comment:
To avoid accessing this underscored property outside of the getter, maybe we can convert this to using the `asObservable` pattern the other outputs are using?

Reply:
Hmmm. our typing isn't even right here. I converted to use the pattern - using a `Subject` since the existing `EventEmitter` didn't hold a value.